src/Microsoft.DotNet.Watcher.Tools/DotNetWatchOuter.targets

```diff
@@ -0,0 +1,35 @@
+<Project
+  xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <Target Name="GenerateWatchList"
+          DependsOnTargets="_CollectWatchProjects;_CollectWatchItems;_WriteGeneratedWatchList"
```

Comment:
Nope. It's only invoked on an inner build. I'm going to rename to `_CoreCollectWatchItems` to make that more clear.